### DISCUSSION TITLE
Issue with setting multiple small projects on different routes of a domain

### DISCUSSION THREAD

**Discussion by MarcoAntolini** [vercel/community#4628] [Help]
### Summary

I have a domain where the main page is my personal website.
I also have some small side projects i wanted to host on my domain and wanted them to be accessible like this: marcoantolini.com/dso-database

### Steps to Reproduce

I added a `vercel.json` file to my main website project with this content:
```
{
	"rewrites": [
		{
			"source": "/dso-database",
			"destination": "https://dso-database-dun.vercel.app"
		},
		{
			"source": "/dso-database/:path*",
			"destination": "https://dso-database-dun.vercel.app/:path*"
		}
	]
}
```
Even though a guide found on the internet suggested to set the destination same as source: `/dso-database`... but tried that and i was getting ERROR 404.
Also added a CNAME to my main project with name `dso-database` and value `dso-database-dun.vercel.app`.
This way works but if i visit marcoantolini.com/dso-database i get a page without css. Also i wanted to have a different favicon for this website and i added it like this:
```
<head>
	<link rel="icon" href="favicon.ico" />
</head>
```
but this still shows the main website favicon.

If somebody knows how to fix these issues, or a different way to make it work the way i want.. i am open to suggestions 😄

**Comment by amyegan** [comment]
Hey @MarcoAntolini! I recommend handling rewrites with your framework if it provides a way. It looks like your site uses Next 13. If that's the case, you should using try the `rewrites` key in `next.config.js`

https://nextjs.org/docs/app/api-reference/next-config-js/rewrites

**Reply by MarcoAntolini** [reply]
Hey @amyegan thanks for the reply.
I also tried as you said using `next.config.js` and setting there the rewrites:
```
/** @type {import('next').NextConfig} */
const nextConfig = {
	async rewrites() {
		return [
			{
				source: "/dso-database",
				destination: "https://dso-database-dun.vercel.app",
			},
			{
				source: "/dso-database/:path*",
				destination: "https://dso-database-dun.vercel.app/:path*",
			},
		];
	},
};

module.exports = nextConfig;
```

But the issue is still the same, when i navigate to marcoantolini.com/dso-database i get a plain html page without css and favicon.
![Screenshot_1](https://github.com/vercel/community/assets/93138565/f15f0956-9f18-498c-94bc-e6faad24d905)
![Screenshot_2](https://github.com/vercel/community/assets/93138565/3ed3ba13-cd93-4041-b1d6-9302fca697e6)

In the pics above it's not visible but on the vercel link i get the right favicon while at marcoantolini.com/dso-database i get the favicon of marcoantolini.com
